brands/amenity/bank.json

```
@@ -6214,6 +6229,19 @@
       "name:en": "VTB Bank"
     }
   },
+  "amenity/bank|ВТБ~Украина": {
```

Again parenthesis here (and add something to the other ВТБ)
```suggestion
  "amenity/bank|ВТБ~(Украина)": {
```